src/test/java/harness/utils/CleanupTestUserWorkspaces.java

```diff
@@ -77,8 +77,8 @@ private static void deleteWorkspaces(TestUsers testUser, boolean isDryRun) throw
               "workspace", "delete", "--workspace=" + workspace.id, "--quiet");
           System.out.println(
               "Cleaned up workspace: id=" + workspace.id + ", testuser=" + testUser.email);
-          deletedWorkspaces.add(workspace.id);
         }
+        deletedWorkspaces.add(workspace.id);
```

Comment:
what's the net effect here?

Reply:
In the script to cleanup test users' workspaces, I report out at the end the list of deleted and failed-to-delete workspaces. This script also has a `dryRun` flag that will not actually do the deletion, but will still print out the lists. Before this change, I wasn't adding "successful" workspaces to the list of deleted ones if the `dryRun` flag was set.